CHANGES.md

```diff
@@ -3,6 +3,10 @@
 
 - Add `(deps <deps>)` in ctype field (#5346, @bobot)
 
+- Add `(foreign_archives ((path <path>)(mode whole))` for ensuring that all the
+  archives in included in the `.cmxs` otherwise only the symbols accessed from
+  the OCaml library are linked ( @bobot )
+
```

Comment:
```suggestion
- Add `(foreign_archives ((path <path>)(mode whole))` to ensure that all the
  archives are included in the `.cmxs`, otherwise only the symbols accessed from
  the OCaml library are linked ( @bobot )

```